aws/resource_aws_appsync_schema.go

```diff
@@ -0,0 +1,87 @@
+package aws
+
+import (
+	"errors"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/appsync"
+	"github.com/hashicorp/terraform/helper/schema"
+	"time"
+)
+
+func resourceAwsAppsyncSchema() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAppsyncSchemaPut,
+		Read:   resourceAwsAppsyncSchemaNil,
```

Comment:
Is it possible to read this information via the `GetIntrospectionSchema` API call? One of Terraform's design tenets is that it can detect configuration drift. If its not possible to read this information back from the API, this may not be a good candidate for management in Terraform.

Reply:
The introspection schema is not the same blob that is sent to StartSchemaCreation.  These are two different objects.  Schema must be present in order to create resolvers.  Setting up the schema outside of Terraform would mean either setting up the API outside of Terraform before hand or setting up resolvers outside of Terraform afterwards.  Neither of these options is ideal.